screen/input.go

```diff
@@ -379,7 +404,7 @@ func (screen *GameScreen) HandleThirdClassInputKeys(input termbox.Event) bool {
 		}
 		if input.Key == termbox.KeyEnter {
 			switch screen.user.GetLocation() {
-			case loud.MARKET, loud.SHOP, loud.FOREST:
+			case loud.HOME, loud.MARKET, loud.SHOP, loud.FOREST:
 				switch screen.scrStatus {
```

Comment:
can we extract this switch case into a different function as we are far too deep into conditions and that makes this code less readable

Reply:
ok